Makefile

```diff
@@ -237,7 +230,7 @@ generate-go: $(GOBINDATA) ## Runs Go related generate targets
 	$(MAKE) generate-go-core
 	$(MAKE) generate-go-kubeadm-bootstrap
 	$(MAKE) generate-go-kubeadm-control-plane
-	$(MAKE) generate-go-operator
+	$(MAKE) -C $(OPERATOR_DIR) generate-go-operator
```

Comment:
I think we should not call the operator targets by default.
Let's call them explicitly, like we are doing for CAPD

Reply:
I thought the operator is a bit different, don't we plan to ship it the same way as `kubeadm` stuff?

Reply:
This makes sense IMO, and it will help identify early issues during manifest generation, when generated resources will overlap between CAPI and CAPIO. CI should handle this part and prevent from PRs being merged if this task fails.